CHANGELOG.md

```diff
@@ -52,6 +52,8 @@ README.md to use the newest tag with new release
 
 - Timeout to wait for cluster health after topology editing
   renamed from `edit_topology_timeout` to `edit_topology_healthy_timeout`
+- `cartridge_cluster_cookie` now is required only for `configure_instance` and
+  `restart_instance` steps
```

Comment:
For alive_not_expelled_instance too

Reply:
No.

Reply:
But you add cluster cookie to module facts of this step in #309....

Reply:
Let's remove cookie from module vars..

Reply:
Yes, that was a mistake.
I will remove it in separated PR, ok?